querybook/webapp/components/DataElement/DataElementDescription.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import {
+    DataElementAssociationType,
+    IDataElementAssociation,
+} from 'const/dataElement';
+import { KeyContentDisplay } from 'ui/KeyContentDisplay/KeyContentDisplay';
+import { AccentText } from 'ui/StyledText/StyledText';
+
+export const DataElementDescription = ({
+    dataElementAssociation,
+}: {
+    dataElementAssociation: IDataElementAssociation;
+}) => {
+    const valueDataElement = dataElementAssociation.value;
+    const keyDataElement = dataElementAssociation.key;
+
+    const valueName =
+        typeof valueDataElement === 'object'
+            ? valueDataElement.name
+            : valueDataElement;
+    const keyName =
+        typeof keyDataElement === 'object'
+            ? keyDataElement.name
+            : keyDataElement;
+
+    const valueDescription =
+        typeof valueDataElement === 'object'
+            ? valueDataElement.description
+            : null;
+    const keyDescription =
+        typeof keyDataElement === 'object' ? keyDataElement.description : null;
+
+    const mapDescrptionDOM = (
+        <div>
+            <KeyContentDisplay keyString={keyName}>
+                <AccentText>{keyDescription}</AccentText>
+            </KeyContentDisplay>
+            <KeyContentDisplay keyString={valueName}>
+                <AccentText>{valueDescription}</AccentText>
+            </KeyContentDisplay>
+        </div>
+    );
+
+    const listDescriptionDOM = (
+        <div>
+            <KeyContentDisplay keyString={valueName}>
+                <AccentText>{valueDescription}</AccentText>
+            </KeyContentDisplay>
+        </div>
+    );
```

Comment:
may be its not necessary to set these constants always.   Can this be a function and return appropriate response depending on type ?
For ex:  majority will be non-composite elements.  So its not necessary to always define and compute, map & list description DOM's


const descriptionDom = (type) => {
      if noncomposite, return <div>
      if array -> just get valueDataElement.description
      etc ?
}